configureWorkspace/config-utils.ts

```diff
@@ -9,6 +9,18 @@ import { IAzureQuickPickItem, IAzureUserInput } from 'vscode-azureextensionui';
 import { ext } from "../extensionVariables";
 import { Platform, PlatformOS } from '../utils/platform';
 
+export type OS = 'Windows' | 'Linux';
```

Comment:
Please remove these and instead use the ones from platform.ts (where they were moved to)

Reply:
Thanks, done. The PlatformOS type from platform.ts was already being used so that shouldn't have still been there.

Reply:
This applies to the extra Platform enum as well...  Thx.

Reply:
D'oh! Fixed.